### PR TITLE
CDS-48540-Added Helm OCI limitation and info about API call to call an OCI image

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
Added Helm OCI limitation and info about API call to call an OCI image.

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by thisrohangupta — APPROVED** [review]

**Review by deepakbolangady — DISMISSED** [review]
The new text looks good. Thanks for checking the titles for style. That correction has, however, caused discrepancies in capitalization between the titles and body text (for example, Environment and Artifact). No need to change anything immediately, but it would be good to change when you can.

File: docs/continuous-delivery/cd-advanced/cd-helm-category/deploy-helm-charts.md [hunk 1]
```diff
@@ -130,10 +132,10 @@ You can also use a local Helm chart if you are deploying the same Helm chart and
...
-  In some cases, such as when using public manifests or Helm charts, you cannot add the annotation.When you enable **Skip Resource Versioning**, Harness will not perform versioning of ConfigMaps and Secrets for the resource.If you have enabled **Skip Resource Versioning** for a few deployments and then disable it, Harness will start versioning ConfigMaps and Secrets.
-- **Helm Command Flags**: You can use Helm command flags to extend the Helm commands that Harness runs when deploying your Helm chart.Harness will run Helm-specific Helm commands and their flags as part of preprocessing. All the commands you select are run before `helm install/upgrade`.
+  You can also select **Expression** and use [Harness expressions](../../../platform/12_Variables-and-Expressions/harness-variables.md) in this setting. The resolved expression must be the name of a Values YAML file in the chart. For example, you could create a stage variable for **values4.yaml** named **qa** and then reference it in **Values YAML** like this: `<+stage.variables.qa>`.
+- **Skip Resource Versioning**: By default, Harness versions configmaps and secrets deployed into Kubernetes clusters.
```
Comment (deepakbolangady) [inline docs/continuous-delivery/cd-advanced/cd-helm-category/deploy-helm-charts.md:136]: "Secrets" can be lowercase but ConfigMaps might need to be in uppercase (except in commands, I would think). I would defer to @michaelcretzman for advice.

**Review by deepakbolangady — APPROVED** [review]
Approving but I would like you to check with MIchael on the capitalization of one term I've commented on. Thanks.

File: docs/continuous-delivery/cd-advanced/cd-helm-category/deploy-helm-charts.md (hunk 1 above)
Comment (michaelcretzman) [reply docs/continuous-delivery/cd-advanced/cd-helm-category/deploy-helm-charts.md:136]: @deepakbolangady yes, ConfigMaps is a Kubernetes object and is always spelled "ConfigMaps".

Comment (roshnisarangadharan) [reply docs/continuous-delivery/cd-advanced/cd-helm-category/deploy-helm-charts.md]: changed back to ConfigMaps

**Review by deepakbolangady — APPROVED** [review]
Checked the capitalization of ConfigMaps. LGTM.

**Pull request merged**